lighthouse-core/audits/themed-omnibox.js

```diff
@@ -8,6 +8,16 @@
 const MultiCheckAudit = require('./multi-check-audit.js');
 const ManifestValues = require('../computed/manifest-values.js');
 const cssParsers = require('cssstyle/lib/parsers');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  title: 'Sets an address-bar theme color',
```

Comment:
`address-bar` -> `address bar` ?

Reply:
> `address-bar` -> `address bar` ?

"address bar" is a phrasal adjective, so should be hyphenated (hyphenating prevents a temporary miscue that "address bar" is the noun here, and prevents misparsing as e.g. "address bar-theme color")

Reply:
I don't see anything online that uses `address-bar` instead of `address bar`, that's why I pointed it out. What if we remove the adjective entirely (nouns as adjectives are weird anyways) - `Sets theme color for the address bar`

Reply:
I like this, much more clear, esp. when I read the dev page, this makes a lot more sense to be something like `theme color for address bar`.